server/src/main.rs

```diff
@@ -34,6 +34,9 @@ enum Args {
 
         #[structopt(name = "PROPOLIS_IP:PORT", parse(try_from_str))]
         propolis_addr: SocketAddr,
+
+        #[structopt(name = "VNC_IP:PORT", parse(try_from_str))]
```

Comment:
Would it make sense to provide a default for this to make the new flag optional?

Reply:
+1, it would be nice not to have to specify this every time.